### DISCUSSION TITLE
如何监听到单元格的点击事件？

### DISCUSSION THREAD

**Discussion by shiguoqian** [dream-num/univer#3727] [Q&A | 常见问题]
能否给个提示？谢谢！

**Comment by jikkai** [comment]
Univer 设计了一套命令系统，你可以通过监听命令来实现绝大多数需求，所以建议先了解一下命令系统的基本概念：https://univer.ai/zh-CN/guides/sheet/getting-started/concepts#%E5%91%BD%E4%BB%A4
